packages/backend/src/document/file-response.util.spec.ts

```
@@ -129,6 +131,37 @@ describe('fileResponseHeaders', () => {
     }
   });
 
+  // Uploads now keep the extension in the title, so the key-based re-append
+  // must not double it — while still covering rows stored before that change.
+  it('does not double an extension the title already carries', () => {
+    expect(
+      fileResponseHeaders('file', 'application/zip', 'bundle.zip', `${UUID}.zip`)
```

ESLint

[error] Replace `'file',·'application/zip',·'bundle.zip',·`${UUID}.zip`)` with `⏎········'file',⏎········'application/zip',⏎········'bundle.zip',⏎········`${UUID}.zip`,`

(prettier/prettier)